xunit.runner.visualstudio.testadapter/VsTestRunner.cs

```diff
@@ -307,9 +309,13 @@ static bool IsXunitTestAssembly(string assemblyFileName)
             if (platformAssemblies.Contains(Path.GetFileName(assemblyFileName)))
                 return false;
 
+#if DOTNET_CORE
+            return true;
```

Comment:
This doesn't seem right. It implies that every assembly in your solution is a test assembly. Is there really no way to check for references like there was back in DNX days?